helm/hpcc/templates/dfuserver.yaml

```diff
@@ -0,0 +1,64 @@
+{{- define "hpcc.dfuServerConfigMap" -}}
+apiVersion: v1
+metadata:
+  name: {{ .me.name }}-configmap
+data:
+  {{ .me.name }}.yaml: |
+    version: 1.0
+    dfuserver:
+{{ toYaml (omit .me "logging") | indent 6 }}
+{{- include "hpcc.generateLoggingConfig" . | indent 6 }}
+{{ include "hpcc.generateVaultConfig" (dict "root" .root "categories" .categories ) | indent 6 }}
+    global:
+{{ include "hpcc.generateGlobalConfigMap" .root | indent 6 }}
+{{- end -}}
+
+{{ range $.Values.dfuserver -}}
+{{- if not .disabled -}}
+{{- $categories := list "system" "storage" -}}
+{{- $commonCtx := dict "root" $ "me" . "categories" $categories }}
+{{- $configSHA := include "hpcc.dfuServerConfigMap" $commonCtx | sha256sum }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .name | quote }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: {{ .name | quote }}
+  template:
+    metadata:
+      labels:
+        run: {{ .name | quote }}
+        accessDali: "yes"
+      annotations:
+        checksum/config: {{ $configSHA }}
+    spec:
+      serviceAccountName: "hpcc-default"
+      initContainers:
+        {{- include "hpcc.checkDataMount" $commonCtx | indent 6 }}
+      containers:
+      - name: {{ .name | quote }}
+        workingDir: /var/lib/HPCCSystems
+        command: [ dfuserver ]
+        args: [
+                {{ include "hpcc.configArg" . }},
+                {{ include "hpcc.daliArg" $ }}
```

Comment:
related to comment re. usage should be changed to --daliServers.
These includes generate --<property> <value>